src/include/daos.h

```diff
@@ -24,6 +24,7 @@
 #include <daos_pool.h>
 #include <daos_mgmt.h>
 #include <daos_security.h>
+#include <daos_pipeline.h>
```

Comment:
let's not include this here now since this is not implemented yet.

Reply:
ok

Reply:
changed